docs/UserGuide.md

```diff
@@ -79,6 +79,29 @@ Examples:
 3. Deletes the program with ID 3 in Pawbook.<br>
    Command: `delete program p/3`
 
+### Edit Command
+
+Edits a dog/owner/program from Pawbook.
+name, Set<Session> sessionSet, Set<Tag> tags
```

Comment:
this line is copy pasted from somewhere?

Reply:
I just wanted to make it easier for me to reference the attributes of a program when writing the DG. Forgot to remove it, thx!